Automattic/jetpack
# tools: Fix PHP max version check in check-development-environment.sh

#### Changes proposed in this Pull Request:

The MAX_PHP_VERSION constant is like "8.0", which should accept any
8.0.x version. Adjust the check to make this happen.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Install PHP 8.0.8.
* Run `tools/check-development-environment.sh`. There should be no message about "PHP at /usr/bin/php is version 8.0.8. We've only tested with PHP up to 8.0."
